README.md

```diff
@@ -73,51 +73,76 @@ NOTE, Ideally, we could provision wordpress-trunk into VVV via https://github.co
+    # If not in shell...
     $ vagrant ssh
+    ```
+    In shell...
+    ```bash
     $ sudo mkdir -p /usr/share/php/phpunit
     $ cd /usr/share/php/phpunit
     $ sudo composer require --dev phpunit/phpunit ^7 --update-with-all-dependencies
     $ sudo ln -sf /usr/share/php/phpunit/vendor/bin/phpunit /usr/bin/phpunit 
     ```
 1. Get the WP Test Suite
     ```bash
+    # If not in shell...
     $ vagrant ssh
-    # As of Sept 2021 pmc-unit-test is compatable with WP 5.8
-    # This is due to https://core.trac.wordpress.org/changeset/51559, as pmc-unit-test is not yet compatable with yoast/phpunit-polyfills
+    ```
+    In shell...
+    ```bash
+    # As of Sept 2021 pmc-unit-test is compatible with WP 5.8
+    # This is due to https://core.trac.wordpress.org/changeset/51559, as pmc-unit-test is not yet compatible with yoast/phpunit-polyfills
+    $ cd ~/
     $ git clone --depth 1 --branch 5.8.1 git@github.com:WordPress/wordpress-develop.git
     ```
 1. Copy the WP Test Suite per site
     ```bash
+    # If not in shell...
     $ vagrant ssh
+    ```
+    In shell...
+    ```bash
     $ cp -r ~/wordpress-develop/tests/ /srv/www/sportico-com/public_html/
     ```
 1. Create `wp-tests-config.php` per site
     ```bash
+    # If not in shell...
     $ vagrant ssh
+    ```
+    In shell...
+    ```bash
     $ cp ~/wordpress-develop/wp-tests-config-sample.php /srv/www/sportico-com/public_html/wp-tests-config.php
     ```
-    1. change line 4 to `define( 'ABSPATH', dirname( __FILE__ ) . '/' );`
-    1. Configure `DB_*` named constants: NOTE the DB_NAME should match your provisioned site (see wp-config.php)
+    1. open the file you copied `wp-tests-config.php` in your editor / IDE of choice.
+    2. change line 4 to `define( 'ABSPATH', dirname( __FILE__ ) . '/' );`
+    3. Configure `DB_*` named constants: NOTE the DB_NAME should match your provisioned site (see wp-config.php)
         ```
         define( 'DB_NAME', 'sportico-com' );
         define( 'DB_USER', 'wp' );
         define( 'DB_PASSWORD', 'wp' );
         define( 'DB_HOST', 'localhost' );
         ```
-    1. Add the following constants
+    4. Add the following constants
         ```
         define( 'PMC_IS_VIP_GO_SITE', true );
         define( 'VIP_GO_APP_ENVIRONMENT', 'development' );
         ```
 1. Run tests 
     1. Note, we must tell PHPUnit where our test bootstraps are located. Note, this must be done each time you SSH into vagrant (See below PHPStorm docs to automate this). Note, change `sportico-com` to the site you're testing within.
-    1. Note, if xdebug is enabled your tests will run VERY slowly. See https://varyingvagrantvagrants.org/docs/en-US/references/xdebug/ Only enable xdebug while testing if you wish to step-through debug your tests.
-    ```
+    2. Note, if xdebug is enabled your tests will run VERY slowly. See https://varyingvagrantvagrants.org/docs/en-US/references/xdebug/ Only enable xdebug while testing if you wish to step-through debug your tests.
```

Comment:
Don't you always want to enable xdebug for the code coverage report?

Reply:
Yes, I can add that too. Good catch!